scripts/globals/mobskills/Corrosive_ooze.lua

```diff
@@ -0,0 +1,32 @@
+---------------------------------------------------
+--  Corrosive Ooze
+--  Family: Slugs
+--  Description: Deals water damage to an enemy. Additional Effect: Attack Down and Defense Down.
+--  Type: Magical
+--  Utsusemi/Blink absorb: Ignores shadows 
+--  Range: Radial
+--  Notes:
+---------------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+---------------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+    local typeEffectOne = EFFECT_ATTACK_DOWN;
+    local typeEffectTwo = EFFECT_DEFENSE_DOWN;
+    local duration = 120;
+
+    MobStatusEffectMove(mob, target, typeEffectOne, 15, 0, duration);
+    MobStatusEffectMove(mob, target, typeEffectTwo, 15, 0, duration);
+
+    local dmgmod = 1;
+    local info = MobMagicalMove(mob,target,skill,mob:getWeaponDmg()*4.2,ELE_WATER,dmgmod,TP_NO_EFFECT);

```

Comment:
mob:getWeaponDmg()*4.2wat

define it as a var above and stick it in the crap (im out of creative names right now so good luck)